ui/scripts/docfy-md.js

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+/* eslint-disable */
+// run this script the ui directory:
+// yarn docfy-md some-component
+//
+// or if the docs are for a component in an in-repo-addon or an engine:
+// yarn docfy-md some-component name-of-engine
```

Comment:
I wonder if we should add this to the index section of the docs. I think it would be useful to have these commands handy when creating a new component and its documentation!

Reply:
Please ignore my comment - I just noticed it on the "How to write component docs" section!

Reply:
On a similar note, I'm always a fan of having a "cheatsheet" section 🙈 but that's an improvement!